findContact/FindContactModal.js

```diff
@@ -131,10 +149,59 @@ class FindContactModal extends React.Component {
     );
   }
 
+  addContacts = () => {
+    const contactList = Object.values(pickBy(this.state.checkedContactsMap));
+
+    this.props.save(contactList);
+  }
+
+  toggleItem = (contact) => {
+    const { id } = contact;
+
+    this.setState(({ checkedContactsMap }) => {
+      const newContact = checkedContactsMap[id] ? null : contact;
+
+      return {
+        checkedContactsMap: {
+          ...checkedContactsMap,
+          [id]: newContact,
+        },
+        isAllChecked: false,
+      };
+    });
+  }
+
+  toggleAll = () => {
+    const contacts = get(this.props.resources, 'records.records', []);
+
+    this.setState((state) => {
```

Comment:
please check docs for correct `setState()` usage https://reactjs.org/docs/react-component.html#setstate
`this.setState((state, props) => {`

Reply:
something similar https://github.com/folio-org/ui-orders/blob/master/src/components/CheckIn/withCheckboxes.js